deploy/maps/action/configmap.action.application.kappnav.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kappnav.actions.application.kappnav
```

Comment:
Probably needs to be something like: name: kappnav.actions.application.{{.Name}} to ensure that it matches the Application CR generated by the operator.

Reply:
I disagree.  This is an instance-specific action config map.  The kappnav Application CR hard codes the name as kappnav.  So the config map name should never be anything other than kappnav.actions.application.kappnav.  We have no reason to ever change Application CR name.

Reply:
Except that it's not hard-coded:

```
	kappnavName := &metav1.ObjectMeta{
		Name:      instance.GetName(),
		Namespace: instance.GetNamespace(),
	}
```

You're deriving the name from the Kappnav CR's name which could in theory be anything.

Reply:
Oh ok.  Didn't realize I left it that way.  Works, but uses whatever the kappnav instance name is.  I changed it to this: 

	kappnavName := &metav1.ObjectMeta{
		Name:      "kappnav",
		Namespace: instance.GetNamespace(),
	}